test/CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
```

Comment:
`CMakeLists.txt.in`? what does `in` mean?

Reply:
it is input file parsed by CMake and modified depending on what is needed - provided by GTest